docs/15_installation.md

```diff
@@ -40,26 +41,36 @@ Running the latest development version
 
 For testing purposes, you can run the latest versions of the client
 and server directly from a clone of
-the [git repository](https://github.com/FreeAndFair/ColoradoRLA).
+the [git repository](https://github.com/FreeAndFair/ColoradoRLA). One
+must configure the database as discussed previously and run both the
+client and the server to test the CORLA system.
 
 For the server, follow the directions
 in [`server/README-ECLIPSE`](../server/README-ECLIPSE.md) to install
 Eclipse, and use Eclipse's Run menu.
 
-Running the system can be accomplished by running the application
-directly (as in `java -jar
-../server/eclipse-project/target/colorado_rla-0.7.0-shaded.jar`) or
-installing the system in a webserver with a Java container.
+Building and packaging the server is accomplished by running `mvn
+package` from the `../server/eclipse-project` directory, as in
+```
+cd server/eclipse-project
+mvn package
+```
 
-To test the client in a standalone manner (rather than from the
-server), you'll need to
-get [Node.js and npm](https://www.npmjs.com/get-npm). Next, run:
+Running the server can also be accomplished by running the application
+directly (as in `java -jar
+../server/eclipse-project/target/colorado_rla-VERSION-shaded.jar`, for
+some specific *VERSION* number) or installing the system in a
```

Comment:
You keep saying this about a webserver with a Java container. I'm not sure I believe it's true.